packages/salesforcedx-vscode-apex/src/commands/forceLaunchApexReplayDebuggerWithCurrentFile.ts

```diff
@@ -81,13 +93,11 @@ async function getApexTestClassName(
   }
 
   await testOutlineProvider.refresh();
-  let testClassName = testOutlineProvider.getTestClassName(sourceUri);
-  // This is a little bizarre.  Intellisense is reporting that getTestClassName() returns a string,
-  // but it actually it returns string | undefined.  Well, regardless, since flushFilePath() takes
-  // a string (and guards against empty strings) using the Non-null assertion operator
-  // (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator)
-  // fixes the issue.
-  testClassName = fileUtils.flushFilePath(testClassName || '');
+
+  const filePath = sourceUri.toString();
+  // filePath is in the format of "file:///Users/{user-name}/{path-to-apex-file.cls}"
+
+  const testClassName = testOutlineProvider.getTestClassName(filePath);
```

Comment:
We no no long use `getTestClassName()`, and instead, within `getTestClassName() it converts the paths to lower case for the comparison.

I believe Windows isn't able to have multiple files with different casing in the same location (eg `fooBar.txt` an `foobar.txt`)

Linux (I think) does support this and one can have files with different casings t the same time (in the same location), but I thin the chances of someone having both `myapexclass.cls` and `MyApexClass.cls` in the same location are pretty slim.

And with Mac OSX, even though its file system is based in Linux, its UI (the Finder) prevents one from naming files with different casing in the same location.

Reply:
So what we're talking about here is a case-sensitive file system.  All OS's have the ability to have case sensitivity turned on for the file system (it's more complicated then that for windows in that it's a more recently supported thing and can be enabled on the folder level :cringe: [source](https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity).  It's been a while but last I recall salesforce doesn't enforce case sensitivity in the org, so I think we're clear to assume that there will never be two files with the same name but different cases. 

I did a quick google to see if I could find the official SF doc on this and found a few external hits but nothing official.  General agreement seems to be that apex class name are case insensitive so we're good to go.  I also verified that if I tried to create a new class with the same name but different casing it errored 👍